environments/pendulum.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, mass, length, g, q=None, p=None):
         self.mass = mass
         self.length = length
         self.g = g
-        super().__init__(q=q, p=p)
+        super(Pendulum, self).__init__(q=q, p=p)
```

Comment:
Why do you call `super `using `Pendulum` and `self`? Does this needs changes in the `Environment` class?

Reply:
Sorry, I think its not necessary. I'll remove it